packages/api/src/models/sensorReadingModel.js

```diff
@@ -87,6 +87,7 @@ class SensorReading extends Model {
     return await SensorReading.query()
       .select('*')
       .joinRaw('JOIN sensor ON sensor_reading.location_id::uuid = sensor.location_id')
```

Comment:
```suggestion
      .joinRaw('JOIN sensor ON sensor_reading.location_id = sensor.location_id')
```
`sensor_reading.location_id` is type `uuid`, no reason for this to be type casted either

Reply:
corrected that as well. Thanks!